packages/components/src/internal/components/user/GroupsList.tsx

```diff
@@ -10,7 +10,7 @@ export const MembersList: FC<Props> = memo(props => {
     return (
         <>
             <hr className="principal-hr" />
-            <div className="principal-detail-label">Member of:</div>
+            <div className="principal-detail-label">Member of</div>
```

Comment:
one more colon to remove. if you go to Admin > Permissions > expand a role that has a group assigned to it > click that group. The right hand Group Details panel still shows a colon after "Members:"
<img width="701" alt="Screen Shot 2022-10-31 at 9 54 20 AM" src="https://user-images.githubusercontent.com/6411206/199038053-aa9d9c9b-6dda-4956-9fd1-406f026a139e.png">

Reply:
Indeed, and the fields in the group panel above also have colons after them.